src/resolve_reports

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from collections import defaultdict
+import csv
+from os import environ
+
+from bs4 import BeautifulSoup
+
+from logic import (
+    determine_timestamp,
+    get_options,
+    get_date_from_filename,
+    uri_schema_doi,
+)
+from utils import doi_to_id, set_token_auth
+
+
+XREF_DOI_PREFIX = environ['XREF_DOI_PREFIX']
+
+
+ARGS = [
+    {
+        'val': '--measure',
+        'dest': 'measure',
+        'action': 'store',
+        'default': None,
+        'help': 'Measure URI, e.g. '
+                'https://metrics.operas-eu.org/citations/crossref/v1',
+        'required': True,
+    }, {
+        'val': '--add-headers',
+        'dest': 'add_headers',
+        'action': 'store_true',
+        'default': [],
+        'help': 'Whether to print report headers',
+        'required': False,
+    }, {
+        'val': '--in_file',
+        'dest': 'in_file',
+        'action': 'store',
+        'help': 'File to read Crossref citation data from (XML format).',
+        'required': True,
```

Comment:
If this is `required`, is it worth specifying a default?

Reply:
I'd avoid it tbh